test/test_train_mp_imagenet.py

```diff
@@ -76,11 +91,17 @@
 
 DEFAULT_KWARGS = dict(
     batch_size=128,
-    test_set_batch_size=64,
+    test_set_batch_size=128,
     num_epochs=18,
     momentum=0.9,
     lr=0.1,
     target_accuracy=0.0,
+    persistent_workers=True,
+    prefetch_factor=32,
```

Comment:
You should probably note that these configurations are enable best perf results on, say, v4-8. This way users know that for other hardware configs, they probably need to make adjustments.

Reply:
I'd be in favor of creating a map data structure that holds all keys needed to get best perf results. Then, pass the map value to the appropriate fields throughout the script.